docs/src/input.md

```diff
@@ -184,6 +184,9 @@ await page.CheckAsync("text=XL");
 - [`method: ElementHandle.check`]
 - [`method: ElementHandle.isChecked`]
 - [`method: ElementHandle.uncheck`]
```

Comment:
```suggestion
```

Let's remove the ElementHandle variants while you are here as Locators are best-practice in most use-cases.